README.md

```diff
@@ -69,14 +69,20 @@ fetch('/api/crc-pdf-generator/v1/generate', {
   },
 
   body: JSON.stringify({
-    // service and template params are mandatory
-    service: 'ros',
-    template: 'executiveReport',
-    // ... any other config options accepted by your template
-  }),
-})
-  // If you are using fetch, don't forget to handle the response.ok === false branch before calling response.blob()!
-  .then((response) => response.blob())
+      // service and template params are mandatory
+      service: 'ros',
+      template: 'executiveReport',
+      // ... any other config options accepted by your template
+    }),
+  })
+  .then(async (response) => {
```

Comment:
There are cases where using async/await within promise callbacks can swallow errors. I don't think we want to promote that in a README